Tests/Runner/RunnerTest.php

```diff
@@ -669,7 +669,7 @@ public function testExecutorReadLegacyState()
                     'import json',
                     'with open("/data/in/state.json", "r") as state_file_read:',
                     '   data = json.load(state_file_read)',
-                    '   assert data["foo"] == "bar", json.dumps(data)'
+                    '   assert data == {}, json.dumps(data)'
```

Comment:
ten test by se mel celej smazat ne?

ale mimochodem me to privadi na myslenku, jestli by na tom state nemel bejt config valiadator co zkontroluje, ze tam je ten korenovej uzel

Reply:
když založíš konfiguraci, tak je state prázdnej.

Reply:
no tady ale do nej prave davas neprazdnej state :) tenhle funkcionalni test bych vyhodil a resil bych to v StateFileTest.php